app/src/main/res/values/colors.xml

```diff
@@ -38,6 +38,7 @@
     <color name="colorOnSecondaryLight">@color/colorSecondaryDark</color>
     <color name="colorOnSecondaryDark">@color/white</color>
 
+    <color name="colorDownloaded">#ddeeff</color>
```

Comment:
Use a more general name so that this color can also be used on other places in the app.